### PR TITLE
fscrawler.bat added a CD to move to the appropriate directory

### PR DESCRIPTION
**Pull request by CircuitGuy**
Issue #1080 uncovered that fscrawler.bat is sensitive to the pwd
  when the batch file is called. Removed that dependency by CDing
  to the expected start directory at the beginning of the bat call

### CODE REVIEW

**Review by dadoonet — COMMENTED** [review]
Would it be possible to store somewhere the current PATH before cdiing into fscrawler dir and after the execution, come back to this dir?

**Comment by CircuitGuy** [comment]
Yeah. Should be easy.

**Comment by dadoonet** [comment]
> Yeah. Should be easy.

Awesome! I'll merge your changes once it's updated 😉

**Comment by CircuitGuy** [comment]
@dadoonet Warning that I did NOT run through a single loop, but tested PUSHD here and verified POPD in a different script.

I'm not sure if the Linux implementation has the same issue or not. Haven't installed this in Linux, but probably will in the next month or two

**Comment by CircuitGuy** [comment]
Actually - please hold off on this PR. There might be a glitch when creating a new job with this modification present. I decided to test the -loop 1, created a new job, and it seems to never get past "job [test2] does not exist. Do you want to create it." prompt.

I'll check into this in the next 24 hours and get back to you.

**Comment by CircuitGuy** [comment]
@dadoonet Sorry - all good. I tested the PR with a single loop and it works as expected.

I missed an escape slash in the YAML settings file. The program response to the invalid file is to say that the job doesn't exist.

**Review by dadoonet — APPROVED** [review]

**Pull request merged**
